src/mame/heathkit/h89.cpp

```diff
@@ -598,6 +598,9 @@ static void tlb_options(device_slot_interface &device)
 static void sigma_tlb_options(device_slot_interface &device)
 {
 	device.option_add("igc", HEATH_IGC);
+	device.option_add("igc_super19", HEATH_IGC_SUPER19);
```

Comment:
If the only difference here is the ROM, this should just use BIOS options.   (Yes, slot devices can have BIOS options - for an extreme example, look at bus/isa/xtide.cpp, which has 24+ ROMs selectable).

Reply:
There are some additional difference that make that hard. The other ROMs all change the definitions of what the dip switch settings mean. The UltraROM has an addition 2k of RAM for off-screen memory. And both the UltraROM and Watzman ROM had different keyboard rom than the original. There are 2 versions of the Watzman ROM, and they are currently using BIOS options.